src/data/defaults.txt

```diff
@@ -1222,6 +1222,7 @@ user	schoolOfHardKnocksDiplomaCost	1000
 user	schoolSwagger	0
 user	scrapbookCharges	0
 user	screencappedMonster	
+user	scriptCascadingMenus	false
```

Comment:
The previous paradigm was that the default was cascading menus built from the file system and all of the control was opting in or out of scriptMRU.  A new preference is fine since if I understand the Control-K changes (which I have no personal interest in and so have not tried yet) they are now the new default.  But we should check to make sure that if a user sets both the menus and the MRU they get a consistent experience - one or the other - and not an unintended hybrid.  My preference GUI work, when finished, could probably be extended or repurposed to handle either or but not both.  We may have to go to a preference whatGoesBelowTheLoadScript option that has three possible values - CK, Cascade or MRU and make scriptMRULength strictly the length and not the control.  Anyway thanks for doing this.

Reply:
I thought about having one control for "what goes below Load Script" but I didn't want to interfere with your pending Preference work. :)

As coded,

if ScriptMRULength <= 0 && scriptCascadingMenus is false, you get nothing
if scriptMRULength > 0, the list of files shown is ONLY from the MRU list
otherwise (scriptCascadingMenus is true), the list of files is from the "scripts" directory